engine/src/main/java/org/terasology/recording/RecordAndReplaySerializer.java

```diff
@@ -119,7 +117,7 @@ private void serializeFileAmount(Gson gson, String recordingPath) {
             writer.close();
             logger.info("File Amount Serialization completed!");
         } catch (Exception e) {
-            logger.error("Error while serializing file amount:", e);
+            logger.error("Error while serializing file amount", e);
```

Comment:
Unless absolutely needed, in the future please refrain from making such small irrelevant changes: the removal of the semi-colon in the error log makes no difference whatsoever.